webpack/assets/javascripts/react_app/components/hosts/storage/vmware/StorageContainer.consts.js

```diff
@@ -0,0 +1,8 @@
+export const MaxDisksPerController = 15;
+export const MaxControllers = 4;
+export const InitialSCSIKey = 1000;
+export const getDiskModeTypes = () => [{
+    'persistent': __('Independent - Persistent')},
```

Comment:
In addition, this data should come from the ruby model. :-)

When mounting the component, you can access the data like this (untested):

```
# Simple hash
controller_types: compute_resource.disk_mode_types
# Array of hashes (or object in js-speech)
controller_types: compute_resource.disk_mode_types.map {|k, v| {k => v}}
# Might also work, gives you an array of arrays
controller_types: compute_resource.disk_mode_types.to_a
```

You can use the same for the controller_types. Map is the same as in JS. Iterates over an object (ruby object) and returns an array.